content/releases/v2.0.0/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Announcing Trento Version 2.0.0"
+date: 2023-05-04T16:21:00+02:00
+hideLastModified: true
+showInMenu: false
+summary: "Available beginning of May’23, Trento 2.0.0 brings major changes to the architecture of the solution..."
+releases: ["v2"]
+layout: test
+image: "trento-release-2-0-0-hero@2x.png"
+# no need for the "summary" parameter as it is not displayed in any previews
+---
+![Trento Release Version 2.0.0](trento-release-2-0-0-hero@2x.png)
+
+Available beginning of May’23, Trento 2.0.0 brings major changes to the architecture of the solution and therefore it will be **necessary an update on both the server** and the agents for those customers that are already using Trento. ​
+
+The main change in the new version is a brand-new SSH-less checks engine, but it also adds VMware to the list of known platforms and provides versioned external APIs for both the web and engine components.
+
+![Trento Checks Engine](trento-checks-engine@2x.png)
+
+# SSH-less Checks Engine
+
+The old engine required an SSH connection from the server to the cluster nodes to execute the checks, which were nothing but Ansible playbooks. With the new checks engine the SSH connection is no longer necessary, which **helps improve the security** of the entire architect. Moving forward there will be only one secured communication channel from the architecture to the server and there will be no need to maintain an SSH key specific for Trento nor an SSH user with elevated access to execute the Ansible playbooks.​
+
+In the new checks engine, that we internally call wanda, the execution of a check is split in two stages. On one side, the agent is responsible for gathering the required data, we call them facts. The facts are sent by the agent to the checks engine which in turn performs their evaluation. This division of responsibilities translates in a reduced footprint on the SAP host and a much better performance.​
+
+The new checks engine comes with a new Domain Specific Language where checks are represented by yaml files with a human readable structure.
+
+![Trento Domain Specific Language - YAML](trento-check-yaml@2x.png)
```

Comment:
just as a one-shot comment for the future: we could have used markdown code blocks instead of screenshotting code, but hey, let's leave it as it is this time.

Reply:
@stefanotorresi I could style the code block to something like this. What do you think?
<img width="541" alt="Screenshot 2023-05-08 at 14 19 47" src="https://user-images.githubusercontent.com/40714533/236822340-591afa79-8603-44a4-882b-2de213c2becd.png">

Reply:
@jagabomb yeah, sure, don't bash your head against this too much atm, just use whatever code style the current hugo theme provides.

Reply:
Ok let me try make these changes and then I think I am all good.